backend/fms_core/template_importer/importers/project_link_sample.py

```diff
@@ -16,6 +16,8 @@ def import_template_inner(self):
         for row_id, row_data in enumerate(project_link_samples_sheet.rows):
             project = {
                 'name': str_cast_and_normalize(row_data['Project Name']),
+                'study_letter': str_cast_and_normalize(row_data['Study']),
+                'study_start': str_cast_and_normalize(row_data['Workflow Step Start']) if row_data['Workflow Step Start'] else None,
```

Comment:
Since this is used both to insert and remove maybe we should rename that column and variables using "step order" instead of "step/study start".